create-proxy.sh

```diff
@@ -30,11 +33,17 @@ fi
 echo "Creating container [${NAME}] using image [${CONTAINER_IMAGE}:${CONTAINER_VERSION}]."
 
 if [ -e "${PSK_FILE}" ]; then
-  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\"; ${START_CMD}"
-else
-  COMMAND="${START_CMD}"
+  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\";"
+if [ -e "${CA_FILE}" ]; then
+  COMMAND="chown zabbix:zabbix \"/var/lib/${CA_FILE}\"; chmod 600 \"/var/lib/${CA_FILE}\";"
+if [ -e "${KEY_FILE}" ]; then
+  COMMAND="chown zabbix:zabbix \"/var/lib/${KEY_FILE}\"; chmod 600 \"/var/lib/${KEY_FILE}\";"
+if [ -e "${CERT_FILE}" ]; then
+  COMMAND="chown zabbix:zabbix \"/var/lib/${CERT_FILE}\"; chmod 600 \"/var/lib/${CERT_FILE}\";"
 fi
 
+COMMAND="${START_CMD}"
```

Comment:
This has been originally in else statement so that if there is not chown to do, it just runs that start command